Airports/EGPG/SMR/regions.txt

```diff
@@ -0,0 +1,182 @@
+; Start Cumbernauld (EGPG) Regions
+; - BG
+REGIONNAME Cumbernauld
+488718 N055.58.26.189 W003.58.56.335
+N055.58.17.697 W003.58.51.722
+N055.58.27.016 W003.58.09.969
+N055.58.29.631 W003.58.05.174
+N055.58.35.104 W003.58.07.464
+N055.58.26.675 W003.58.54.158
+N055.58.26.294 W003.58.56.384
+; - Taxiway
+REGIONNAME Cumbernauld
+6383724 N055.58.32.132 W003.58.09.687
+N055.58.32.106 W003.58.09.615
+N055.58.32.042 W003.58.09.570
+N055.58.32.005 W003.58.09.550
+N055.58.31.160 W003.58.09.757
+N055.58.31.108 W003.58.09.765
+N055.58.31.077 W003.58.09.801
+N055.58.31.061 W003.58.09.853
+N055.58.30.947 W003.58.10.620
+N055.58.30.824 W003.58.11.247
+N055.58.30.685 W003.58.11.532
+N055.58.30.556 W003.58.11.690
+N055.58.30.458 W003.58.11.725
+N055.58.30.415 W003.58.11.722
+N055.58.29.592 W003.58.11.117
+N055.58.29.615 W003.58.10.995
+N055.58.29.442 W003.58.10.784
+N055.58.29.269 W003.58.11.060
+N055.58.28.075 W003.58.10.388
+N055.58.28.284 W003.58.09.380
+N055.58.28.981 W003.58.09.763
+N055.58.29.149 W003.58.08.591
+N055.58.30.474 W003.58.09.251
+N055.58.30.702 W003.58.09.201
+N055.58.30.920 W003.58.09.233
+N055.58.30.972 W003.58.09.176
+N055.58.31.177 W003.58.09.298
+N055.58.32.253 W003.58.09.080
+N055.58.32.134 W003.58.09.745
+; - Dissued
```

Comment:
As with the geo file, please name these so that they can be easily found in the future.